packages/synapse-react-client/src/components/SynapseHomepageV2/SynapseHomepageV2.tsx

```diff
@@ -0,0 +1,514 @@
+  fontWeight: 300,
+  fontSize: {
+    xs: '48px',
+    md: '72px',
+  },
+  lineHeight: {
+    xs: '120%',
+    md: '82px',
+  },
+  color: 'white',
+}
+const sidePadding: SxProps = {
+  pl: {
+    xs: '15px',
+    sm: '50px',
+  },
+  pr: {
+    xs: '15px',
+    sm: '50px',
+  },
+}
+
+export const SynapseHomepageV2: React.FunctionComponent = () => {
+  const theme = useTheme()
+  const isDesktopView = useMediaQuery(theme.breakpoints.up('lg'))
+
+  //optimization - prioritize loading above-the-fold content (delay loading below the fold)
+  const { ref, inView } = useInView({ triggerOnce: true })
+  return (
+    <Box>
+      <SynapseHomepageNavBar />
+      <Box sx={{ position: 'relative', pb: '80px' }}>
+        <Box
+          sx={{
+            position: 'absolute',
+            background: `url(${backgroundInlineJpeg}) lightgray 50% / cover no-repeat`,
+            minHeight: '600px',
+            width: '100%',
+            height: '100%',
+            content: "''",
+            top: 0,
+            right: 0,
+            bottom: 0,
+            left: 0,
+            zIndex: -1,
+            clipPath: 'polygon(0 0, 100% 0, 100% 65%, 0% 100%)',
+          }}
+        >
+          {' '}
+        </Box>
+      </Box>
+
+      <Box
+        sx={{
+          textAlign: 'center',
+          p: {
+            xs: '20px',
+            lg: '80px 0px 0px 0px',
+          },
+          minHeight: {
+            xs: '270px',
+            md: undefined,
+          },
+        }}
+      >
+        <Typography variant="headline1" sx={titleSx}>
+          Explore the data
+        </Typography>
+        <Typography variant="headline1" sx={titleSx}>
+          behind&nbsp;
+          <TypeAnimation
+            sequence={[
+              // Same substring at the start will only be typed out once, initially
+              'the next cure',
+              3000,
+              'the next diagnostic',
+              3000,
+              'the next preventive therapy',
+              3000,
+            ]}
+            wrapper="span"
+            speed={20}
+            repeat={Infinity}
+            style={{
+              color: '#B5D3CE',
+            }}
+          />
+        </Typography>
+      </Box>
+
+      {/* Search */}
+      <Box
+        sx={{
+          pt: '70px',
+          maxWidth: '600px',
+          m: 'auto',
+        }}
+      >
+        <SynapseHomepageSearch sourceTable={searchAutocompleteTable} />
+      </Box>
+      <Box
+        sx={{
+          display: 'flex',
+          gap: '10px',
+          alignItems: 'center',
+          justifyContent: 'center',
+          mt: '30px',
+          flexWrap: 'wrap',
+        }}
+      >
+        {/* Hard-coded popular searches because these Chips are above the fold and immediately visible. Any delay showing these chips is a problem.
+        The hope is that these "popularSearches" will not change much over time, since they represent the types of data in the platform.
+        */}
```

Comment:
Later, if these need to be dynamic & fast, we could look into serving the home page as a separate app with SSR